### DISCUSSION TITLE
SStudent developer packs offers

### DISCUSSION THREAD

**Discussion by Ntshangase** [community/community#47350] [GitHub Education] [answered]
How do I use the developer pack offers, for example the intro into web dev is there but I can't see button to accept the package or individually install the packages.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @Ntshangase 

`Intro to web dev` is a collection of a few things you can get access to.

Here is the link for you to see:

https://education.github.com/experiences/intro_to_web_dev


https://user-images.githubusercontent.com/47532703/218763789-aa0c5655-0908-4e41-94ca-fe52721b7d5c.mp4
